pkgs/getsops/sops/registry.yaml

```diff
@@ -1,13 +1,13 @@
 packages:
   - type: github_release
-    repo_owner: mozilla
+    repo_owner: getsops
     repo_name: sops
```

Comment:
```suggestion
    repo_owner: getsops
    repo_name: sops
    aliases:
      - name: mozilla/sops
```

We should set `aliases` to keep the compability.

https://aquaproj.github.io/docs/reference/registry-config/aliases/